src/services/Item/index.ts

```diff
@@ -0,0 +1,25 @@
+import Axios from 'axios';
+import { IncomingMessage } from 'http';
+
+import { listConfig, actionConfig } from './config';
+import { ItemListParams, ItemActionRequestParams } from '@src/types';
+import { IItem } from '@src/interfaces';
+
+const list = async (
+  params: ItemListParams & { userId?: number },
+  req?: IncomingMessage
+): Promise<IItem[]> => Axios(listConfig(params, req)).then((res) => res.data);
+
+const action = async (
```

Comment:
```suggestion
const action = async (
```

action은 이 기능에 대해 명확히 설명하지 못 하는 것 같아요.
tag라는 네이밍이 더 좋을 것 같아요.

Reply:
e418508